src/components/RoundedBackground.tsx

```diff
@@ -1,13 +1,13 @@
 import { heightPercentageToDP as hp } from 'react-native-responsive-screen'
 import styled from 'styled-components/native'
 
-const RoundedBackground = styled.View<{ color?: string; height?: string }>`
+const RoundedBackground = styled.View<{ color?: string; height?: string; bottom?: string }>`
```

Comment:
```suggestion
const RoundedBackground = styled.View<{ color?: string; height?: string; marginBottom?: string }>`
```

Reply:
I had no problem at the beginning, but now it keeps crushing everytime

Reply:
I didn't change anything except the line in styling and the line in component

Reply:
fixed